src/loc/statement/StatementOfFactsTemplateEN.tsx

```diff
@@ -27,6 +27,12 @@ export default function StatementOfFactsTemplateEN(props: Props) {
                 <div>{ props.pathModel.polkadotAddress }</div>
             </div>
 
+            { props.pathModel.imageSrc.startsWith("data:image") &&
```

Comment:
Why this condition? We might as well pass a regular URL and it should work the same, no?

Reply:
Adapted: https://github.com/logion-network/logion-wallet/compare/1aa682cc039e90c541a64b542695a4a2368bc21d..89d5ef7b968f05b934e1e5c45ae8bc6f466bc6a3